server/rooms.ts

```diff
@@ -1686,6 +1687,7 @@ export const Rooms = {
 	},
 	createChatRoom(roomid: RoomID, title: string, options: AnyObject) {
 		if (Rooms.rooms.has(roomid)) throw new Error(`Room ${roomid} already exists`);
+		if (!options.creationTime) options.creationTime = Date.now();
```

Comment:
This should be filled in the same place other options are filled in (inside the `BasicChatRoom` constructor).